lib/animate_icons.dart

```diff
@@ -36,6 +36,9 @@ class AnimateIcons extends StatefulWidget {
     /// The duration for which the animation runs
     this.duration,
 
+    /// The amplitude of the rotation in degrees
+    this.amplitude,
```

Comment:
I would put the default value here
`this.amplitude = 180.0`

Reply:
I wrote it like this to be consistent with the rest of the code. As you can see, all the other not required parameters are passed in the constructor as nullable, and the null values are handled in the `_AnimateIconsState` class with the `??` operator.